app/services/birth_record_service.rb

```diff
@@ -6,10 +6,30 @@
 # rather than using the ActiveRecord queries directly.
 class BirthRecordService
   # Search for a single BirthRecord matching the supplied parameters hash
-  def self.query(params)
+  def self.query(params, case_insensitive_keys = self.case_insensitive_keys)
```

Comment:
1. should this be `def self.query(params, case_insensitive_keys: self.case_insensitive_keys)`
2. probably could do with a test for being sensitive on different keys if you're going to have the option, or removing the option

Reply:
👍 👍